woocommerce/sensei-certificates
# Ensure font files can be accessed when plugin is copied to another site

Fixes #96.

This PR essentially reverts #124, but then tweaks the value of the `$ttffile` variable in `dejavusanscondensed.mtx.php` so that it's not [hard-coded](https://github.com/woocommerce/sensei-certificates/pull/124/files#diff-08b5bb8aad6feb4db2763a7f3ced10f9L16).

I believe the original issue was happening when users copied the plugin files from one server to another, which would invalidate the value of the `$ttffile` variable.

## Testing Instructions
- On a brand new site (site 1), install/activate Sensei LMS from WP.org and the `master` version of this plugin.
- Activate the Twenty Nineteen theme.
- Preview a certificate template and ensure that it works.
- Spin up another new site (site 2), install/activate Sensei LMS again, but COPY `sensei-certificates` from site 1 to site 2. Rename the folder to something else (e.g. `sensei-certificates-2`).
- Activate the plugin.
- Activate the Twenty Nineteen theme on site 2.
- Preview a certificate template on site 2. It shouldn't work.
- Switch to this branch and copy the plugin files into the `sensei-certificates-2` folder on site 2.
- Preview the certificate template again. It should work now.
